app/shared/footer/FooterContent.js

```diff
@@ -41,7 +45,7 @@ function FooterContent({ t }) {
     case 'VANTAA': {
       return (
         <Grid>
-          <Row>
+          <Row className="flex-container">
```

Comment:
You should never put custom classNames and styles for Row and Column components. Rather you can add a div or other wrapping element inside Column.